biorbd_optim/optimal_control_program.py

```diff
@@ -459,11 +461,137 @@ def add_plot(self, fig_name, update_function, phase_number=-1, **parameters):
 
         nlp["plot"][plot_name] = custom_plot
 
+    def export_eocar_ode_model(self):
+        from acados_template import AcadosModel
+        from casadi import MX, Function, external, SX, vertcat, sin, cos
+
+
+        m = biorbd.Model(self.nlp[0]['model'].path().relativePath().to_string())
+        model_name = "model_name"
+
+        # Declare model variables
+        x = MX.sym('x', m.nbQ() * 2)
+        u = MX.sym('u', m.nbQ())
+        xdot = MX.sym('dx', m.nbQ() * 2)
+
+        f = Function('f', [x, u],
+                     [vertcat(x[m.nbQ():], m.ForwardDynamics(x[:m.nbQ()], x[m.nbQ():], u).to_mx())]).expand()
+        x = SX.sym('x', m.nbQ() * 2)
+        u = SX.sym('u', m.nbQ())
+        xdot = SX.sym('dx', m.nbQ() * 2)
+        f_expl = f(x, u)
+        f_impl = xdot - f_expl
+
+        model = AcadosModel()
+        model.f_impl_expr = f_impl
+        model.f_expl_expr = f_expl
+        model.x = x
+        model.xdot = xdot
+        model.u = u
+        # model.z = z
+        model.p = []
+        model.name = model_name
+
+        return model
+
+    def __prepare_acados(self):
```

Comment:
As a second thought, can you create a file `acados_interface.py` which will call function inside instead of putting this in opc.py? For symmetry, if you could move the solve() part which solely relate to ipopt in a `ipopt_interface.py` that would be great!